Makefile

```diff
@@ -6,8 +6,7 @@ templating:
 	@go fmt atlas/commands/bootstrap/templates/template-bindata.go
 
 test:
-	@echo $(thing)
-	@go test ./...
+	@go test -v ./...
 
 test-with-integration:
-	@export e2e=true && go test ./...
+	@export e2e=true && go test -v ./...
```

Comment:
we'll eventually want to modify this more to get test coverage output as well, but not for now

Reply:
Agreed. I've been holding off on coverage because I want to refactor `main_test.go` first. The current iteration of `main_test.go` register any increase in test coverage, so the coverage looks abysmal.

I think a better solution is to create a `boostrap_test.go` with a set that does the following: 

1. Modifies the `init-app` flags on a per-test basis
2. Calls `bootstrap.Run` to bootstrap the test application(s)
3. For each application that gets bootstrapped, do the HTTP stuff to ensure it responds with a 200

I already messed around with this when I tried concurrently bootstrapping applications.